### DISCUSSION TITLE
Welcome to creative Discussions!

### DISCUSSION THREAD

**Discussion by mairin** [RedHatOfficial/creative#14] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kalpeshmarathe** [comment]
Hello My name is kalpesh and i m prsuing b.tech degree in computer science i m currently in my second year , i have knowledge about programming languages and some front-end technologies like HTML CSS BOOTSTRAP etc. i want to contribute in opensource will you plzz Help me or Suggest me that how can i contribute or  how can i inhance my those industry relevant skills . and will you plzz suggest me some resources 

here is my portfolio link
https://kalpeshmarathe.studio/

**Reply by notklaatu** [reply]
Hi @kalpeshmarathe

I think one of the best ways to get started contributing to open source is to start using open source. You'll inevitably find things that are asking for improvements. For instance, you might start with http://fedoraproject.org first, and then once you get the hang of that, check out https://whatcanidoforfedora.org to find a community with opportunities for contribution.

Another avenue is right here on Github. Browse through projects you use or that interest you, look at their Issues for something you might be able to solve, and start solving problems.

It's as easy and also as confusing as that. There are no bosses in open source. There are project maintainers who have opinions on what is acceptable or not within their own project, so you ought to respect their experience and oversight, but it's entirely up to you to find places to contribute, and then to make contributions. It can seem overwhelming, but it's perfectly acceptable to start small.  Posting answers to people with problems you know how to solve is a valid contribution to  open source. Posting your photography on Unsplash or Flickr under a [Creative Commons](https://opensource.com/article/20/1/what-creative-commons) license is a valid contribution to open source. [Start contributing](https://opensource.com/life/13/4/ten-ways-participate-open-source), and you're a contributor!

I hope this helps.
